src/modules/jobs/models/tech-load.model.ts

```diff
@@ -0,0 +1,399 @@
+  }
+}
+
+interface JobData {
+  accountNumber: number,
+  firstName: string,
+  lastName: string,
+  location: Location,
+  appointment: string,
+  email: string,
+  phone: number,
+  jobType: string,
+  transportType: string,
+  history: History[],
+  facilities: Facility[],
+  services: Services,
+}
+
+interface Facility {
+  heading: string,
+  address: string,
+  cable?: number,
+  pair?: number,
+  port?: number,
+}
+
+interface Services {
+  equipment: string[];
+  internetSpeed?: number;
+  potsNumber?: number;
+  voipNumbers?: number[];
+  uversePackage?: string;
+}
+
+interface History {
+  title: string,
+  date: string,
+  info: string[]
+}
+
+interface Location {
+  streetAddress: string,
+  state: string,
+  city: string,
+  zip?: number,
```

Comment:
Put optional at end of interface.

Reply:
I moved lat/long to the top so I could keep city/state/zip together but have zip at the bottom.